Sources/Purchasing/StoreKit1/StoreKit1Wrapper.swift

```diff
@@ -133,6 +133,10 @@ extension StoreKit1Wrapper: SKPaymentTransactionObserver {
     func paymentQueue(_ queue: SKPaymentQueue, updatedTransactions transactions: [SKPaymentTransaction]) {
         guard let delegate = self.delegate else { return }
 
+        if transactions.count >= 100 {
```

Comment:
let's extract 100 here into a constant so it's not a magic number in the middle of the code